easybuild/easyconfigs/d/Delft3D/Delft3D-4.04.01-foss-2022a-FLOW.eb

```diff
@@ -0,0 +1,84 @@
+easyblock = 'ConfigureMake'
+
+name = 'Delft3D'
+_revision = '142633'
+version = '4.04.01'
+versionsuffix = '-FLOW'
+
+homepage = 'https://oss.deltares.nl/web/delft3d'
+description = """
+Simulation of multi-dimensional hydrodynamic flows and transport phenomena, including
+sediments. Delft3D-FLOW is part of Delft3D 4.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'usempi': True}
+
+download_instructions = """
+There is no tarball provided; it needs to be created through an svn checkout:
+  1. svn checkout https://svn.oss.deltares.nl/repos/delft3d/trunk@%s
+  2. tar -czvf %%(name)s-%%(version)s.tgz trunk
+""" % _revision
+
+sources = [SOURCE_TGZ]
+patches = ['Delft3D-4.04.01_use-external-esmf.patch']
+checksums = [
+    # Delft3D-4.04.01.tgz
+    None,
+    # Delft3D-4.04.01_use-external-esmf.patch'
+    '86f127cf3193c6fc9256491d917051db91e695718631d1b221b111499e0f432b',
+]
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('libtool', '2.4.7'),
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+    ('Ruby', '3.0.5'),
+    ('Autotools', '20220317'),
+    ('M4', '1.4.19'),
+]
+
+dependencies = [
+    ('netCDF-Fortran', '4.6.0'),
+    ('METIS', '5.1.0'),
+    ('PETSc', '3.17.4'),
+    ('expat', '2.4.8'),
+    ('ESMF', '8.3.0'),
```

Comment:
ESMF is not required, can be removed as none of the binaries or libraries depend on it.
i guess it is there for the other Delft3D components.

Reply:
@smoors No longer relevant thanks to the changes made via https://github.com/WilleBell/easybuild-easyconfigs/pull/3?

Reply:
it's the nice fix of @jfgrimm  in https://github.com/WilleBell/easybuild-easyconfigs/pull/3 that introduced this dependency, which i think is not necessary.

Reply:
no idea whether that would be part of the workflow, but it did get installed. I'll have a look at patching it to use our ESMF dep

Reply:
hm indeed, better to keep it in then

Reply:
@smoors @WilleBell should be fixed in https://github.com/WilleBell/easybuild-easyconfigs/pull/4